webextensions/manifest/page_action.json

```diff
@@ -72,7 +72,7 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": true,
+                "version_added": "≤54",
```

Comment:
@queengooborg the PR that added these three items appears to have been created on February 27, 2019, and merged on March 1, 2019. This would make the relevant version 72.